pages/registered/page-registered.htm

```diff
@@ -25,6 +26,6 @@
 <div class="jumbotron row featured text-center">
 	<h4 class="buffer">Continue Shopping or Shop our featured collection</h4>

```

Comment:
Could be useful to link to the "/shop", like this:

`<h4 class="buffer"><a href="/shop">Continue Shopping</a> or Shop from our featured collection</h4>`